files/image_config/system-health/system-health.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Delays SONiC system health monitor until SONiC has started
+
+[Timer]
+OnBootSec=1min 30 sec
```

Comment:
Please don't introduce a timer. Instead can you make sure its event based?

Reply:
@dgsudharsan Done